### ISSUE TITLE
[Feature Request] Apple TV aerial videos as wallpapers

### ISSUE THREAD

**Issue by testxar** [spocky/miproja1#168]
Please consider adding the option to set one of the Apple TV aerial videos as the wallpaper.

Spreadsheet with the links
https://docs.google.com/spreadsheets/d/1bboTohF06r-fafrImTExAPqM9m6h2m2lgJyAkQuYVJI/edit?gid=1684411812#gid=1684411812

Example videos: https://bzamayo.com/watch-all-the-apple-tv-aerial-video-screensavers#5a0745b9f8e451e21fabe93e4463cc76

**Comment by Thorfinn66** [comment]
Why don't you just use Arial Views screensaver?
https://github.com/theothernt/AerialViews

**Comment by peat80** [comment]
://github.com/spocky/projectivy-plugin-wallpaper-overflight/releases

Should be possible now with this addon.

**Comment by spocky** [comment]
Yes, Overflight json list was developped based on this spreadsheet.
